modules/plan/src/main/PayPalClient.scala

```diff
@@ -43,6 +44,9 @@ final private class PayPalClient(
 
   private val patronMonthProductId = "PATRON-MONTH"
 
+  // todo: proper Eq implementation
+  given Eq[Currency] = Eq.fromUniversalEquals
```

Comment:
`Eq.fromUniversalEquals` means we'll just use the old `Currency.equals` as Eq implementation. So the logic should be the same as before. then I don't think we need to do anything more here.